lib/generators/doorkeeper/templates/initializer.rb

```diff
@@ -314,6 +314,23 @@
   #   client.superapp? or resource_owner.admin?
   # end
 
+  # Implement constraints in case you use Client Credentials to authenticate
+  # the introspection endpoint. By default allow a client can only introspect
+  # the token issued to it, otherwise disallow.
+  #
+  # Params:
+  # `token` - the token to be introspected
+  # `client` - the client authorized for introspection endpoint
```

Comment:
Let's add here that `client` is an instance of `Doorkeeper::Application` entity so developers wouldn't need to check what type is it

Reply:
If using `Doorkeeper::Application` entity, should I use the param name to `application` instead of `client`? Because other configuration options is use `client` param name as `Doorkeeper::OAuth::Client` entity.

Reply:
It's OAuth term that we widely use (client). Application is just the concrete entity in our solution. I recommend client as we use it in other places

Reply:
I kept the `client` term and added comments to these params.